manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "oct8ne",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

@ingmoyete, no need to bump the version here. This is done automatically.

```suggestion
  "version": "2.0.5",
```